src/modules/returns/lib/jobs/map-to-json.js

```diff
@@ -87,6 +95,19 @@ const uploadJsonToS3 = (eventId, json) => {
   return s3.upload(jsonFileName, Buffer.from(str, 'utf8'));
 };
 
+const onFailed = async (job, err) => {
+  logger.error(`${JOB_NAME}: Job has failed`, err);
+};
+
+const onComplete = async (job, queueManager) => {
+  // Format and add BullMQ message
+  const { eventId, companyId } = job.data;
+  await queueManager.add(validateReturns.jobName, { eventId, companyId });
+  logger.info(`${JOB_NAME}: Job has completed`);
```

Comment:
As above, Why did you not add these steps in the handler? Is it not better to perform the job functions in the handler rather than complete?